test/e2e_node/e2e_node_suite_test.go

```diff
@@ -135,15 +133,9 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 	framework.AfterReadingAllFlags(&framework.TestContext)
-	if err := e2eskipper.InitFeatureGates(utilfeature.DefaultFeatureGate, featureGates); err != nil {
-		fmt.Fprintf(os.Stderr, "ERROR: initialize feature gates: %v", err)
-		os.Exit(1)
-	}
 
-	if err := services.SetFeatureGatesForInProcessComponents(serviceFeatureGates); err != nil {
-		fmt.Fprintf(os.Stderr, "ERROR: initialize process feature gates for API service: %v", err)
-		os.Exit(1)
-	}
+	initFeatureGates(utilfeature.DefaultMutableFeatureGate, featureGates)
+	initFeatureGates(defaultMutableServiceGate, serviceFeatureGates)
```

Comment:
Can't you move this into a non-test file, like it was in some previous revision of this PR? Then we don't need to add exceptions to the verify script.

Reply:
Done!
linter is happy now :)

Thank you and @mmiranda96